### PR TITLE
Test: DocCallout

### LINKED ISSUE #1343: UNIT TESTS - components/DocWrapper/DocCallout
## What

Create unit tests using [Jest](https://jestjs.io/en/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).

Here are a couple examples of how unit tests should be created:

1. [Unit tests for our A component](https://github.com/thenewboston-developers/Website/blob/development/src/components/A/A.test.tsx)
2. [Unit tests for Icon component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Icon/Icon.test.tsx)
3. [Unit tests for Button component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Inputs/Button/Button.test.tsx)

## General Acceptance Criteria for writing Unit tests

1. Should create a `[nameOfComponent].test.tsx` within the same directory of the component.
2. Should have tests that tests every `prop` of the component.
3. If there are any conditional logic within the component, there should be unit tests that accounts for each conditional branches.
4. `npm run test` should be passing.

## Specific Acceptance Criteria

1. make sure you tests for each prop

### PR DESCRIPTION
**Pull request by PrinceRajRoy**
Resolves #1343
Account Number: `be7df48f6d6e58ce4da6a6b0c763d205b63924bf8d18e86bfd6a7e8f5f2a9275`

### CODE REVIEW

File: src/components/DocWrapper/DocCallout/DocCallout.test.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    render(<DocCallout {...baseProps} />);
+
+    expect(screen.getByTestId('DocCallout')).toBeTruthy();
+  });
+
+  it('renders with default className', () => {
+    render(<DocCallout {...baseProps} />);
+
+    expect(screen.getByTestId('DocCallout')).toHaveClass('DocCallout');
+    expect(screen.getByTestId('DocCallout__header')).toHaveClass('DocCallout__header');
```
Comment (jamessspanggg) [inline src/components/DocWrapper/DocCallout/DocCallout.test.tsx:23]: Could you also add a check for the className with modifier (`DocCallout${modifier}`, `DocCallout__header${modifier}`) here?

**Review by jamessspanggg — COMMENTED** [review]
just one nit

File: src/components/DocWrapper/DocCallout/DocCallout.test.tsx (hunk 1 above)
Comment (PrinceRajRoy) [reply src/components/DocWrapper/DocCallout/DocCallout.test.tsx:23]: Yes, sure

**Comment by Rajputusman** [comment]
@PrinceRajRoy It seems like a new account number with your github account. I have sent 2000tnbc on this account number let me know if you want to make this account number linked with github account,

**Comment by PrinceRajRoy** [comment]
@Rajputusman yup, this is the correct one, the last one was a mistake but the transaction was already done, so I let it be. Please update it to this Account No.

**Comment by Rajputusman** [comment]
@PrinceRajRoy Updated Thanks for your cooperation.

**Comment by PrinceRajRoy** [comment]
@Rajputusman just a quick QQ, how long does it take for the transfer? I haven't received this one yet

**Comment by Rajputusman** [comment]
@PrinceRajRoy The coins have been sent to your account for this PR on the given account.

**Pull request merged**